Cargo.toml

```diff
@@ -10,8 +10,6 @@ documentation = "http://ironframework.io/doc/router/index.html"
 keywords = ["iron", "web", "http", "routing", "router"]
 
 [dependencies]
+iron = "~0"
```

Comment:
@untitaker Yes, this is intended for current situation. It will match latest iron until there is an actual breaking change in iron, or the router crate reaches 1.0. 
 If we do have a new iron breaks current API, in 0.4 for example, we can then release a newer router with this been changed to something like `^0.4`